appmap.yml

```diff
@@ -1,6 +1,6 @@
 name: spring-petclinic
 packages:
   - path: org.springframework.samples.petclinic
-    shallow: true
+    shallow: false
```

Comment:
The default config generated by `appmap-java` sets `shallow: true` for all packages, in the interest of produce smaller AppMaps. To see the calls in subpackages (e.g. `org.springframework.samples.petclinic.service.impl`), we need to set `shallow: false` (or remove the `shallow` attribute altogether).